src/logging/json.rs

```diff
@@ -279,7 +282,18 @@ where
             }
 
             for (key, val) in &self.static_fields {
-                ser.serialize_entry(key, val)?;
+                if !self.parse_env_in_static {
+                    ser.serialize_entry(key, val)?;
+                } else {
+                    match val.as_str() {
+                        Some(value) if value.starts_with('$') => {
```

Comment:
This is good, but you can do better. Ideally you should parse every occurrence of '$' followed by variable name within value, and try to exchange it with a value from environment.